bundle/language_en.properties

```diff
@@ -321,6 +323,10 @@ angal.billbrowser.today.btn
 angal.billbrowser.today.btn.key                                                  = T
 angal.groupsbrowser.groupmenu.btn                                                = GroupMenu
 angal.groupsbrowser.groupmenu.btn.key                                            = M
+angal.lab.n.btn                                                                  = N
+angal.lab.p.btn                                                                  = P
+angal.labnew.ip.btn                                                              = IP
+angal.labnew.opd.btn                                                             = OPT
```

Comment:
Fix value to "OPD"

Reply:
I'll fix it but the original value in the bundle was "OPT"

Reply:
we just had chance to fix it 😉 thanks!